src/status_im/contexts/wallet/rpc_data_store/networks.cljs

```diff
@@ -54,7 +54,9 @@
 
 (defn- sort-networks
   [networks]
-  (sort-by (juxt :layer :short-name) networks))
+  ;; Placing the ethereum chains (including testnet) in the top of the list during sorting
+  (sort-by (juxt #(not (contains? networks.config/ethereum-chain-ids (:chain-id %))) :layer :short-name)
```

Comment:
Would be great to isolate the usage of the network config values into functions inside the `networks.core` ns, smth like `ethereum-chain?`. 

```clj
;; .../networks/core.cljs
(defn ethereum-network?
  [network]
  (contains? networks.config/ethereum-chain-ids (:chain-id network)))

;; .../rpc-data-store/networks.cljs
(defn- sort-networks
  [networks]
  (sort-by (juxt (complement networks/ethereum-network?) :layer :short-name) networks))

```

Reply:
Totally agree. I have updated 👍